datahub-web-react/src/app/entity/shared/containers/profile/nav/EntityProfileNavBar.tsx

```diff
@@ -14,21 +16,27 @@ type Props = {
 const AffixWithHeight = styled(Affix)``;
 
 export const EntityProfileNavBar = ({ urn, entityType }: Props) => {
+    const appConfig = useAppConfig();
     const { data: browseData } = useGetBrowsePathsQuery({
         variables: { input: { urn, type: entityType } },
         fetchPolicy: 'cache-first',
     });
     const entityRegistry = useEntityRegistry();
     const isBrowsable = entityRegistry.getBrowseEntityTypes().includes(entityType);
 
+    const { showBrowseV2 } = appConfig.config.featureFlags;
+
     return (
         <AffixWithHeight offsetTop={60}>
-            <ProfileNavBrowsePath
-                urn={urn}
-                type={entityType}
-                breadcrumbLinksEnabled={isBrowsable}
-                path={browseData?.browsePaths?.[0]?.path || []}
-            />
+            {showBrowseV2 && <ProfileNavBrowsePathV2 urn={urn} type={entityType} />}
```

Comment:
okay i think this answers it.. browse V2 is completely independent of the search feature

Reply:
that's correct - could change though!